### DISCUSSION TITLE
user-guide/organize-library

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#194] [文档评论区]
# user-guide/organize-library

Zotero 非官方中文维护小组

https://zotero-chinese.com/user-guide/organize-library

**Comment by Eternal579** [comment]
写的非常好🥰

**Comment by distancehh** [comment]
各位贡献者好 想问下如何快速批量删除（合并）zotero中的重复条目 而不是一个个手动合并 bilibili上有人提过一个插件但是已经很久没有更新 可能已经不兼容了 https://www.bilibili.com/read/cv16419180/

**Reply by northword** [reply]
Z7：https://zotero-chinese.com/plugins/#search=zoplicate

**Comment by tuyahur** [comment]
蹲一个解决方案：完成一篇论文写作后，进行下一项研究内容时，前一篇的参考文献不需要时刻打开，但希望可以在必要时打开查阅

**Reply by 379000** [reply]
您好，请问找到解决方案了吗

**Comment by cquleepf** [comment]
您好，请问 zotero 7，使用shift+左键 自动文献分类时，和直接拖动一样，还是复制到新的分类要怎么排除问题呢？

**Reply by cquleepf** [reply]
我尝试把所有插件都关闭，仍然是这样

**Comment by zhxxl** [comment]
请问怎么样新建多的文库呢？就是我不希望所有东西都在“我的文库”之下，求解答~

**Comment by Athena-Psyer** [comment]
之前好像使用过自动添加标签，但是选择删除自动添加的标签后还是剩下很多我没有添加过的，并没有在右键中找到批量删除的方法

**Reply by Athena-Psyer** [reply]
安装了action-tag的插件，没太学会怎么用hh

**Reply by Athena-Psyer** [reply]
观察后发现很多标签主要内容是一样的，但是有大小写差异也会被分为不同个标签，比较麻烦

**Comment by tangtangqi** [comment]
想请问一下大家右下角删除自动标签一直是灰色的点不了，这个要怎么解决或者有什么代替方法吗？

**Comment by left99** [comment]
谢谢贡献者的教程与维护。
我想请教一下，同一篇文献，在不同分类下，重要性不同。但是软件中似乎只能存在一个评级，请问这个有啥好的解决方案吗，谢谢

**Reply by 379000** [reply]
我也想要知道这个
